compiler/testData/codegen/boxAgainstJava/notNullAssertions/doGenerateParamAssertions.kt

```diff
@@ -15,3 +17,20 @@ public static void runTest(doGenerateParamAssertions a) {
         throw new AssertionError("Fail: IllegalArgumentException expected");
     }
 }
+
+// FILE: B.kt
+import test.doGenerateParamAssertions as C
```

Comment:
Would be nice to rename the class itself now that it's possible. Previously the Java class was named this way because its file was compiled directly by javac, but box tests copy everything to tmpdir with new names, so we can use the short name for convenience

Reply:
Done